NYPLAudiobookToolkit/UI/AudiobookPlayerViewController.swift

```diff
@@ -564,6 +580,17 @@ let SkipTimeInterval: Double = 15
         let logString = "\(#file): Player reported an error. Audiobook: \(bookID)"
         ATLog(.error, logString, error: error)
     }
+    
+    // This needs to be a function because the tintColor is updated after this VC being created.
```

Comment:
it's only computed once if you provide a initialization closure, but I think a non-lazy computed property should do what you want:
```
private var progressViewBackgroundColor: UIColor {
   ......
}
```